com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderUtil.java

```diff
@@ -25,9 +25,50 @@
 import org.codehaus.jackson.map.ObjectMapper;
 
 import com.dynamo.bob.CompileExceptionError;
-
+import com.dynamo.graphics.proto.Graphics.ShaderDesc;
 
 public class ShaderUtil {
+    public static class Common {
```

Comment:
A lot of the shader transpilation uses these functions so just moved them into a separate namespace